explainers/navigation-to-unsigned-bundles.md

```diff
@@ -0,0 +1,363 @@
+that's shared within the same claimed origin within a single bundle, but which
+can't conflict with or attack:
+
+* sites served verifiably from that claimed origin (e.g.
+  `https://publisher.example/page.html?q=query`),
+* other bundles (e.g. `package://https!,,distributor.example,otherpackage.wbn;q=query$https!,,publisher.example/page.html?q=query`), or
+* other origins within the same bundle (e.g. `package://https!,,distributor.example,package.wbn;q=query$https!,,otherpublisher.example/page.html?q=query`).
+
+## Open design questions
+
+* Do we need an "internal redirect" notion for the bundle->primary-URL redirect?
+  See also [whatwg/fetch#576](https://github.com/whatwg/fetch/issues/576).
+
+### Network access
+
+Should untrusted bundles be able to go to the network at all? Should users be
+able to explicitly grant permission?
+
+Under repressive governments, untrusted bundles may be a good way for dissidents
+to share information, but if the government can add their own tracking code to
+the bundles, they may be able to catch people using the banned information.
+
+However, there are also use cases like for sharing book previews, where a
+publisher might want to provide both a free preview and the encrypted rest of
+the book in the same unsigned bundle, while letting users buy a small decryption
+key with an online flow. On the other hand, perhaps that use case should use the
+signed bundles that aren't being proposed in this explainer.
+
+## Security and privacy considerations
+
+* Unsigned bundles served by one origin can't be trusted to provide content for
+  another origin. The [URL design](#urls-for-bundle-components) and the division
+  between trusted and untrusted bundles are designed to keep separate origins
+  safely separate, but this is more than a trivial amount of complexity, and
+  browsers will need to be careful to actually enforce the boundaries.
+
+* It's straightforward for someone serving an unsigned bundle to include a
+  unique ID in the resources within that bundle. If the bundle can then make
+  [network requests](#network-access), especially credentialed network requests,
+  the author can determine a set of people who are connected to eachother in
+  some way. This is detectable if users have a way to inspect two bundles that
+  are supposed to hold the same content, but since the whole point of sharing
+  bundles is to reduce redundant transfers, it's unlikely many users  will
+  actually check this.
+
+  Sites currently gather information on this kind of link sharing by annotating
+  their URLs with unique or semi-unique IDs. These can usually be removed by
+  removing query parameters, but if a significant number of users cleaned their
+  URLs, the tracking could move to path segments.
+
+## Considered alternatives
+
+### Alternate formats considered
+
+There are several existing ways to serialize a web page or website, but all of
+them have problems:
```

Comment:
nit: problems -> shortcomings for the usage described in this explainer ...or something?  (For some particular use cases there can be reasons to prefer them to WBN, I suppose)